src/sgame/sg_main.cpp

```diff
@@ -210,6 +210,7 @@ Cvar::Cvar<int> g_combatCooldown("g_combatCooldown", "team change disallowed unt
 Cvar::Range<Cvar::Cvar<int>> g_debugEntities("g_debugEntities", "entity debug level", Cvar::NONE, 0, -2, 3);
 
 Cvar::Cvar<bool> g_instantBuilding("g_instantBuilding", "cheat mode for building", Cvar::NONE, false);
+Cvar::Cvar<bool> g_cheatIgnoreNobuild("g_cheatIgnoreNobuild", "ignore nobuild area", Cvar::NONE, false);
```

Comment:
`g_instantBuilding` is classified as a regular server option (I think I even saw trem servers with it), but it seems `g_cheatIgnoreBuild` is a cheat :smiley:. I think the easiest is just to mark it as such and then (IIRC, needs testing I guess) you don't need to check for `g_cheat` afterwards.

```suggestion
Cvar::Cvar<bool> g_cheatIgnoreNobuild("g_cheatIgnoreNobuild", "ignore nobuild area", Cvar::CHEAT, false);
```

Reply:
I'm not sure at all if this should be a cheat or not. But it seems i made a mistake here, because i check the value of g_cheats myself when using the cvar's value.

Reply:
Don't think it needs to be Cvar::CHEAT. If an admin wants it, why not :shrug:

Reply:
I agree, did that.